### PR TITLE
Updating Readme for Hacktoberfest contestants

### LINKED ISSUE #4: Make an Appealing README for #Hacktoberfest Contestants
The current README.md file seem to need some improvements to make it more appealing.

### PR DESCRIPTION
**Pull request by kubahasek**
Closes #4

Edited the readme to make it clear the repo is hacktoberfest oriented.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,8 +32,6 @@ See `contributing.md` for ways to get started.
 
 Please adhere to this project's `code of conduct`.
 
-
 ## Authors
```
Comment (SauveJeanLuc) [inline README.md:35]: Can you improve the Authors part ?
Like something creative displaying the profile for every contributor ?

Comment (kubahasek) [reply README.md:35]: Okay, will do!

Comment (kubahasek) [reply README.md:35]: @SauveJeanLuc are you alright with me using the AllContributors bot?

Comment (SauveJeanLuc) [reply README.md:35]: No worries @kubahasek.
You can use it

Comment (kubahasek) [reply README.md:35]: @SauveJeanLuc I'd need you to install it though. I don't have the permissions to do it for this repo. <https://allcontributors.org/docs/en/bot/installation>, here is the link to it.

**Review by akhilsharmaa — APPROVED** [review]

**Comment by akhilsharmaa** [comment]
Good job

**Comment by kubahasek** [comment]
Don't think this was finished, why didn't it get merged?

**Comment by akhilsharmaa** [comment]
There are some Conflicts in merging. Work on that.

**Comment by kubahasek** [comment]
I am waiting for @SauveJeanLuc's reply about adding the AllContributors bot first

File: README.md (hunk 1 above)
Comment (SauveJeanLuc) [reply README.md:35]: @kubahasek.
I've installed it.

**Comment by SauveJeanLuc** [comment]
> I am waiting for @SauveJeanLuc's reply about adding the AllContributors bot first

I've installed it.
Remember to resolve conflicts though.

**Comment by kubahasek** [comment]
@SauveJeanLuc Done!

I would suggest updating CONTRIBUTING.md as people now have to tag the bot and tell it to add them.

**Comment by SauveJeanLuc** [comment]
> @SauveJeanLuc Done!
> 
> I would suggest updating CONTRIBUTING.md as people now have to tag the bot and tell it to add them.

Raise a PR for this.

**Comment by SauveJeanLuc** [comment]
@kubahasek ,
I think we'll need at list a profile picture and username for each contributor in the ReadMe.

image

What do you think ?

File: README.md [hunk 2]
```diff
@@ -1,10 +1,12 @@
-
 # Coffee Tasks
 
 A [Hacktoberfest](https://hacktoberfest.com/) Present 🎁 to React Beginners
 
 ![image](https://user-images.githubusercontent.com/68426990/192261187-84c1029b-686c-4549-a41d-9f23394a98a9.png)
 
+# Created for **HACKTOBERFEST**! Come learn front-end development with us throughout the month of October!
```
Comment (SauveJeanLuc) [inline README.md:8]: I think this section can be ignored for now. Cz

- The message is redundant as it was already stated above and below this section
- The section will shout with a big font

File: README.md [hunk 3]
```diff
@@ -1,10 +1,12 @@
...
+
+# **Description of the project:**
```
Comment (SauveJeanLuc) [inline README.md:10]: This should have smaller font size that project title

**Comment by kubahasek** [comment]
> @kubahasek , I think we'll need at list a profile picture and username for each contributor in the ReadMe.
> 
> image
> 
> What do you think ?

That's correct, here is the guide <https://allcontributors.org/docs/en/bot/usage> for using the bot. Each user can do it on their own.

File: README.md (hunk 3 above)
Comment (kubahasek) [reply README.md:10]: Do you want me to remove it altogether, or just lower the size then?

Comment (SauveJeanLuc) [reply README.md:10]: You can remove `#Created for **HACK ...`
And lower the font size of `#** Description of the project`

**Comment by kubahasek** [comment]
@all-contributors add @kubahasek for docs

**Comment by SauveJeanLuc** [comment]
> @all-contributors add @kubahasek for docs

Great, Let's try this out.

**Comment by SauveJeanLuc** [comment]
@all-contributors add @SauveJeanLuc for maintenance

**Review by SauveJeanLuc — APPROVED** [review]

**Comment by SauveJeanLuc** [comment]
@all-contributors add @SauveJeanLuc for Maintenance

**Pull request merged**